### DISCUSSION TITLE
I'm struggling with click__out event

### DISCUSSION THREAD

**Discussion by michalblaszak** [elimintz/justpy#708] [Q&A]
Basically I'm trying to implement a simple functionality. A button when clicked opens a popup (div). Then when clicked outside the popup I want it to be hidden. I used 'click' event on the button and 'click__out' event on the popup. The problem is that clicking on the button triggers the 'click__out' event too so the popup gets closed immediately.
I tried different approaches with no success. The click__out target attribute doesn't seem to indicate the button.
Any idea how to fix it?

Best regards,
Michał Błaszak

**Comment by WolfgangFahl** [comment]
There is not much traffic here any more given that nicegui has 8x more stars than justpy.

**Reply by dridk** [reply]
Too bad, because I think justPy's API is better.

**Comment by WolfgangFahl** [comment]
see issue #704  - feel free to take the lead in this project
